tests/test_config_files.py

```diff
@@ -19,4 +19,17 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from .version import *  # Generated by sconsUtils
+import pathlib
+import unittest
+
+from lsst.ts import salobj
+
+
+class ConfigTestCase(salobj.BaseConfigTestCase, unittest.TestCase):
+    def setUp(self):
+        self.config_package_root = pathlib.Path(__file__).parents[1]
+
+    def test_OCPS(self):
+        self.check_standard_config_files(sal_name="OCPS",
+                                         module_name="lsst.dm.OCPS",
+                                         config_package_root=self.config_package_root)
```

Comment:
Please add `config_schema="CONFIG_SCHEMA"` because as written this will look for a yaml file.

You will also have to make CONFIG_SCHEMA a public symbol in your package and make it a dict, not a yaml string.